### PR TITLE
Fixes #496 - Added date/time variable explanation

### LINKED ISSUE #496: Add explanation for new date/time formatting in triggers, schedulers, ...
We introduced a new feature to be able to format date/time values, e.g. in the mail that is sent by a trigger (e.g. an auto-responder).

For this, we need another headline in https://admin-docs.zammad.org/en/latest/system/variables.html and a good explanation:

* What is possible?
* What could be used?
* ~~Known limitations~~

We will update this issue here with a proposal.

### PR DESCRIPTION
**Pull request by ralf401**

### CODE REVIEW

File: system/variables.rst [hunk 1]
```diff
@@ -66,4 +66,62 @@ Zammad which you can set in the admin panel under :doc:`/settings/branding`.
...
+     - Use ``%e`` for day without zero-padding
+   * - ``%H``
+     - Hour of day in 24h notation with zero-padding
+     - Use ``%l`` / ``%I`` for hour in 12h notation with ``%p`` or  ``%P``
+   * - ``%I``
+     - Hour of day in 12h notation with zero-padding
+     - Use ``%l`` for hour without zero-padding and ``%p`` / ``%P`` for meridian indicator
+   * - ``%M``
+     - Minute of hour
+     -
```
Comment (fliebe92) [inline system/variables.rst:110]: This could be removed I guess. :D

File: system/variables.rst [hunk 2]
```diff
@@ -66,4 +66,62 @@ Zammad which you can set in the admin panel under :doc:`/settings/branding`.
...
+
+Date and Time Formatting
+------------------------
+
+If you want to use a date and time format which is different from your Zammad
+default (see :doc:`branding </settings/branding>`), you can use ``dt``
+in combination with a date/time variable.
+
+| **Example variable:**
+| ``#{dt(ticket.updated_at, "%A, %Y-%m-%d %H:%M (Europe/Berlin)", "Europe/Berlin")}``
```
Comment (fliebe92) [inline system/variables.rst:79]: TODO: Explain the format string

File: system/variables.rst [hunk 3]
```diff
@@ -66,4 +66,62 @@ Zammad which you can set in the admin panel under :doc:`/settings/branding`.
...
-lead to the output of the original (english) name of the state.
\ No newline at end of file
+lead to the output of the original (english) name of the state.
+
+Date and Time Formatting
+------------------------
+
+If you want to use a date and time format which is different from your Zammad
+default (see :doc:`branding </settings/branding>`), you can use ``dt``
+in combination with a date/time variable.
```
Comment (fliebe92) [inline system/variables.rst:76]: TODO: The `dt` method could be used ...

File: system/variables.rst [hunk 4]
```diff
@@ -66,4 +66,62 @@ Zammad which you can set in the admin panel under :doc:`/settings/branding`.
...
+     - Use ``%l`` / ``%I`` for hour in 12h notation with ``%p`` or  ``%P``
+   * - ``%I``
+     - Hour of day in 12h notation with zero-padding
+     - Use ``%l`` for hour without zero-padding and ``%p`` / ``%P`` for meridian indicator
+   * - ``%M``
+     - Minute of hour
+     -
+   * - ``%S``
+     - Second of hour
+     -
```
Comment (fliebe92) [inline system/variables.rst:113]: TODO: Remove

**Review by fliebe92 — CHANGES_REQUESTED** [review]
Great work, thanks! Only some small changes are required.

**Review by tschaefer — APPROVED** [review]

**Review by fliebe92 — APPROVED** [review]

**Pull request merged**